model/src/main/kotlin/TextLocation.kt

```diff
@@ -38,7 +38,7 @@ data class TextLocation(
     /**
      * The line the text is ending at.
      */
-    val endLine: Int
+    val endLine: Int = startLine
```

Comment:
Add "Defaults to the [startLine]." to the docs.
For a nicer name when using named arguments you could also add a secondary constructor instead:
```
constructor(path: String, line: Int) : this(path, startLine = line, endLine = line)
```